src/dos/dos_programs.cpp

```diff
@@ -68,6 +69,7 @@ void Add_VFiles(const bool add_autoexec)
 #endif
 	PROGRAMS_MakeFile("BOOT.COM", ProgramCreate<BOOT>);
 	PROGRAMS_MakeFile("CHOICE.COM", ProgramCreate<CHOICE>);
+	PROGRAMS_MakeFile("FMPDRV.EXE", FMPDRV_ProgramCreate);
```

Comment:
Consider adding `FMPDRV.EXE` only if ReelMagic emulation is enabled.

Reply:
Added, this is a nice touch, @FeralChild64.

It's a one-way-street though; once we've registered the driver in Z:\, it's there for good (even if the card is disabled after). This simply because we don't have a way to de-register files from the Z:\ (which isn't a big deal).